Eplant/views/eFP/Tooltips/EFPTooltip.tsx

```diff
@@ -102,7 +124,7 @@ function SVGTooltip(props: {
                       </TableCell>
                       <TableCell sx={{ borderBottom: 'none' }}>
                         {Math.log2(
-                          props.tissue.mean / (props.data.control ?? 1)
```

Comment:
I haven't tried running the code but isn't this comma a syntax error?

Reply:
I think that comma is from the auto-formatting from Prettier? It's not a syntax error but I agree it looks a bit weird lol